### PR TITLE
Run with --skipLibCheck

### PR DESCRIPTION
**Pull request by mhegazy**
Makes tests run twice as fast on my machine. also less errors duplication when say lodash has an error, it will be reported once when lodash is checked instead of in every package that depends on lodash

### CODE REVIEW

File: src/harness/externalCompileRunner.ts [hunk 1]
```diff
@@ -74,6 +76,7 @@ abstract class ExternalCompileRunnerBase extends RunnerBase {
                     args.push("--types", types.join(","));
                 }
                 args.push("--noEmit");
+                if (additionalArgs && additionalArgs.length) args.push(...additionalArgs);
```
Comment (DanielRosenwasser) [inline src/harness/externalCompileRunner.ts]: Pretty sure the `.length` check is superfluous.

Comment (DanielRosenwasser) [reply src/harness/externalCompileRunner.ts]: Also, when is `additionalArgs` undefined? You have a default parameter.

**Comment by DanielRosenwasser** [comment]
Looks good, but get Wes or Nathan to check first.

**Comment by mhegazy** [comment]
@weswigham and @sandersn any comments?

**Comment by weswigham** [comment]
`skipLibCheck` skips all declaration file checking.... everything on DT other than tests are declaration files; and important checks (like subclass substitutability) _are_ performed on declaration files (and we want to know if we break them or change their errors!). I'm not sure this is safe to do... It's pretty much tossing away 70% of the suite. `skipDefaultLibCheck` is safe, though - we could definitely do that.

**Comment by mhegazy** [comment]
most of the errors are in tests really. but yes, we are not checkoing the .d.ts files at this point.

**Comment by weswigham** [comment]
I'd think most of the errors are in the tests at this point because keeping the declaration files themselves error-free is comparatively high priority. And these tests are (part of) how we check that.

**Comment by sandersn** [comment]
Errors do show up only in declaration files from time to time. I don't think we should stop checking them.

**Comment by mhegazy** [comment]
Ok then, leaving this one out.

**Comment by weswigham** [comment]
@mhegazy what about `skipDefaultLibCheck`?

**Comment by mhegazy** [comment]
sure.. i can add that for all of them actually.

**Pull request closed without merging**